src/piadm.sh

```diff
@@ -681,32 +679,27 @@ enablepool() {
 		# For now, proceed clobber-style.
 	fi
 
-	output=$(zfs list -H $bootfs 2>&1)
-	if [[ $? -ne 0 ]]; then
+	if ! zfs list -H "$bootfs" 2>&1; then
```

Comment:
Same thing here:  the output needs to go to /dev/null.

Reply:
Fixed